ios_widget_catalog_compare/ios/Runner/OverlayFlutterViewController.swift

```diff
@@ -0,0 +1,190 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import SwiftUI
+
+
+/// The initial `UIViewController` defined in `Main.storyboard` uses this class.
+/// This class overlays a SwiftUI view on top of the Flutter application.
+/// It also communicates with the Flutter code to signal when a new widget must be shown.
+@available(iOS 14.0, *)
+class OverlayFlutterViewController: FlutterViewController, FlutterStreamHandler, ObservableObject {
+  
+  let dropDownButton: UIButton = UIButton(frame: CGRect(x: 13, y: 50, width: 300, height: 40))
+  let disclaimerLabel: UILabel = UILabel()
+  let resetButton: UIButton = UIButton()
+  let xSlider: UISlider = UISlider()
+  let ySlider: UISlider = UISlider()
+  let alphaSlider: UISlider = UISlider()
+  lazy var slidersStackViewHeader: UIStackView = UIStackView(arrangedSubviews: [disclaimerLabel, resetButton])
+  lazy var slidersStackView: UIStackView = UIStackView(arrangedSubviews: [slidersStackViewHeader, xSlider, ySlider, alphaSlider])
+  lazy var swiftUIController: UIViewController = UIHostingController(rootView: OverlaySwiftUIView(controller: self))
+  
+  lazy var overlaySwiftUIView: OverlaySwiftUIView = OverlaySwiftUIView(controller: self)
+  
+  // The overlay's center point is assigned when self.view appears.
+  var originalOverlayCenter: CGPoint = .zero
+  
+  // OverlaySwiftUIView listens to this var's publisher to
+  // present the selected control.
+  @Published var controlKey: String = ""
+  
+  var eventSink: FlutterEventSink?
+  lazy var eventChannel: FlutterEventChannel = FlutterEventChannel(name: "overlay_ios.flutter.io/responder", binaryMessenger: self as! FlutterBinaryMessenger)
```

Comment:
keep line widths at 100 chars to keep them readable on common monitor widths

Reply:
I don't know how to count 100 chars other than eyeballing. But yeah I'll change this

Reply:
it's a bit buried, but there's a config for this (if you're using xcode) 

![Screen Shot 2020-10-08 at 12 16 27 PM](https://user-images.githubusercontent.com/156888/95503370-22c86180-0960-11eb-84b2-cf33f841e620.png)